app/helpers/automated_tests_helper.rb

```diff
@@ -175,19 +206,21 @@ def can_run_test?
   # Export group repository for testing
   def export_repository(group, repo_dest_dir)
     # Create the test framework repository
-    unless File.exists?(MarkusConfigurator.markus_config_automated_tests_repository)
-      FileUtils.mkdir(MarkusConfigurator.markus_config_automated_tests_repository)
+    unless File.exist?(MarkusConfigurator.
+                           markus_config_automated_tests_repository)
+      FileUtils.mkdir(MarkusConfigurator.
+                          markus_config_automated_tests_repository)
     end
 
-    # Delete student's assignment repository if it already exists
+    # Delete student's assignment repository if it already exist
     repo_dir = File.join(MarkusConfigurator.markus_config_automated_tests_repository, group.repo_name)
-    if File.exists?(repo_dir)
+    if File.exist?(repo_dir)
       FileUtils.rm_rf(repo_dir)
     end
 
     return group.repo.export(repo_dest_dir)
-    rescue Exception => e
-      return "#{e.message}"
+  rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?